lib/rules/prefer-user-event.ts

```diff
@@ -88,50 +88,34 @@ export default createTestingLibraryRule<Options, MessageIds>({
 
   create(context, [options], helpers) {
     const { allowedMethods } = options;
-    const sourceCode = context.getSourceCode();
 
     return {
-      ['CallExpression > MemberExpression'](node: TSESTree.MemberExpression) {
-        const util = helpers.findImportedUtilSpecifier('fireEvent');
-        if (!util) {
-          // testing library was imported, but fireEvent was not imported
+      'CallExpression Identifier'(node: TSESTree.Identifier) {
+        if (!helpers.isFireEventMethod(node)) {
```

Comment:
I extracted logic from this rule to `isFireEventMethod` so the rule is really simple now!